node-runner-cli/templates/radix-fullnode-compose.j2

```diff
@@ -0,0 +1,134 @@
+services:
+{% if core_node is defined %}
```

Comment:
This is Jinja template file which gets templated out based on values in config gile